Query.php

```diff
@@ -164,7 +165,9 @@ public function andFilterCompare($name, $value, $defaultOperator = '=')
         } else {
             $op = $defaultOperator;
         }
-
+        if(isset($result['$err'])){
```

Comment:
Yes, you are right, i wanted add this fix to the fetchRowsInternal method, and i haven't method andFilterCompare, and it is strange.